app/controllers/booking_controller.rb

```diff
@@ -1,50 +1,36 @@
 class BookingController < ApplicationController
-
-	get '/bookings/new' do #create new booking
-		if logged_in?
-			erb :'/booking/new'
-		else
-			redirect '/login'
-		end
-	end
-
 	post '/bookings' do
 		booking = Booking.new(
 			time: DateTime.strptime(params[:time], '%Y-%m-%dT%H:%M:%S%z'),
-			user_id: session[:id],
+			user_id: session[:user_id],
 			court: params[:court]
 			)
 
 		if booking.save
-			redirect '/bookings/show'
+			redirect '/bookings'
 		else
 			redirect '/bookings/new'
 		end
 	end
 
-	get '/bookings/show' do
+	get '/bookings' do
 		if logged_in?
-			erb :'/bookings/show'
+			erb :'/bookings/index'
 		else
 			redirect '/login'
 		end
 	end
 
 	get '/bookings/:id/edit' do
 		@booking = Booking.find_by(id: params[:id])
-		if session[:id] == @booking.user.id && @booking.time > DateTime.now
-			if @booking
+		if @booking && session[:user_id] == @booking.user.id && @booking.time > DateTime.now
```

Comment:
I like what you are doing here, but it would be more readable and useful to use your current_user method here instead of session

```ruby 
if @booking && current_user == @booking.user && @booking.time > DateTime.now
```